src/python/install.sh

```diff
@@ -461,13 +467,26 @@ if [ "${INSTALL_JUPYTERLAB}" = "true" ]; then
         exit 1
     fi
 
-    install_user_package jupyterlab
-    install_user_package jupyterlab-git
+    INSTALL_UNDER_ROOT=true;
```

Comment:
```suggestion
    INSTALL_UNDER_ROOT=true
```

Reply:
Fixed in b39c96402c296ff9d158f61741af84d0b07c0003